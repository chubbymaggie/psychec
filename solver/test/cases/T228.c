void f(void* p)
{
    g(p);
}

void f2()
{
    void* p2;
    g2(p2);
}
