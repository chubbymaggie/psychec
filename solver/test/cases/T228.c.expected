/* Forward declarations */

/* Type definitions */

/* Variables and functions */
 int /*<<< orphan*/  g (void*) ;
 int /*<<< orphan*/  g2 (void*) ;

